README.md

```
@@ -0,0 +1,199 @@
+
+Or with [uv](https://docs.astral.sh/uv/):
+
+```bash
+uv add pcp-mcp
+```
+
+## 📋 Requirements
+
+- **Python**: 3.10+
+- **PCP**: Performance Co-Pilot with `pmproxy` running
+  ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)